Code/Engine/ECS/Systems/PlayerControllerSystem.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <Engine/ECS/Registry.h>
+#include <Engine/ECS/Components/TransformComponent.h>
+#include <Engine/InputManager.h>
+
+namespace sad
+{
+	class PlayerControllerSystem
+	{
+	public:
+		PlayerControllerSystem();
+		~PlayerControllerSystem();
+
+		static void Update();
```

Comment:
What do you think about passing a reference to the `EntityWorld&` in our system updates?

Do you know if it's better to pass by reference (and have a reference in the system) or call `ecs::Registry::GetEntityWorld()` each time we want the world in each method of the system?